src/components/SecondaryPanes/Expressions.css

```diff
@@ -6,12 +6,20 @@
   cursor: pointer;
   color: var(--theme-body-color);
   background-color: var(--theme-body-background);
-  line-height: 12px;
+  font-size: 12px;
 }
 
 .input-expression::-webkit-input-placeholder {
   text-align: center;
   font-style: italic;
+  color: var(--theme-body-color);
+}
+
+.input-expression::-moz-placeholder {
```

Comment:
@nchevobbe group them doesn't seem to work. I tried searching for this and got a question on stackoverflow relevant to this [here](http://stackoverflow.com/questions/16982449/why-isnt-it-possible-to-combine-vendor-specific-pseudo-elements-classes-into-on) .